### PR TITLE
Fix enum conversion

### PR DESCRIPTION
**Pull request by jingming**

### CODE REVIEW

File: src/main/java/com/twilio/converter/Promoter.java [hunk 1]
```diff
@@ -33,4 +33,28 @@ public static URI uriFromString(final String url) {
...
+     * @param <T> enum type
+     * @return converted enum if able to convert; null otherwise
+     */
+    public static <T extends Enum<?>> T enumFromString(final String value, final T[] values) {
+        if (value == null) {
+            return null;
+        }
+
+        for (T v : values) {
+            if (v.toString().equalsIgnoreCase(value)) {
```
Comment (efossier) [inline src/main/java/com/twilio/converter/Promoter.java]: No longer need to do the mapping of `-` -> `_`?

Comment (efossier) [inline src/main/java/com/twilio/converter/Promoter.java]: Also the issue we saw this morning was the value coming over the wire had a space ie `toll free` and the enum stripped out the space, should we also do that here?

Comment (jingming) [inline src/main/java/com/twilio/converter/Promoter.java]: This change makes it so that it compares the value of the enum to the string that comes in.

**Comment by efossier** [comment]
👍

**Pull request merged**